fastly/dictionary_info_test.go

```diff
@@ -0,0 +1,59 @@
+package fastly
+
+import (
+	"testing"
+	"time"
+)
+
+func TestClient_GetDictionaryInfo(t *testing.T) {
+	var (
+		nd  *DictionaryInfo
+		err error
+	)
+	record(t, "dictionary_info/get", func(c *Client) {
+		nd, err = c.GetDictionaryInfo(&GetDictionaryInfoInput{
+			ServiceID: testServiceID,
+			Version:   682,
+			ID:        "6mJXuCi2Mf19uHRefZJZBg",
+		})
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !nd.LastUpdated.Equal(time.Date(2019, time.June, 4, 19, 23, 11, 0, time.UTC)) {
+		t.Errorf("bad last_updated: %v", nd.LastUpdated)
+	}
+	if nd.ItemCount != 4 {
+		t.Errorf("bad item_count: %d", nd.ItemCount)
+	}
+	if nd.Digest != "44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a" {
+		t.Errorf("bad digest: %q", nd.Digest)
+	}
+}
```

Comment:
This test is not generic and therefore doesn't work if someone makes changes and needs to create new recordings. The test setup should look the same as in [dictionary_item_test.go](https://github.com/fastly/go-fastly/blob/master/fastly/dictionary_item_test.go). There are a lot of helper functions located in [fastly_test.go](https://github.com/fastly/go-fastly/blob/master/fastly/fastly_test.go).

Your test should do the following:

1. Create a dictionary
2. Create one or more items
3. Call GetDictionaryInfo
4. Assert **only** that ItemCount matches with the sum of the items you have created earlier

Reply:
Thanks for the guidance @philippschulte, and for your patience.

I've updated the tests to reflect what I believe it needs to be, however I can't sort out how to run them to create the new fixtures files.   I've set my `FASTLY_API_KEY` var, but obviously don't have access to the service ID defined in `fastly_test.go`, so I get a 401.

What's the recommendation for that?

Reply:
@jmhobbs we need to document this process! You don't have access to the service in the repository. Here is what you need to do:

Prerequisites:
* a Fastly account with at least one dummy service
* an API token with write access to the Fastly account and your dummy service

Steps:
1. set the `FASTLY_API_KEY` environment variable with your API token
2. add the ID of your dummy service [here](https://github.com/fastly/go-fastly/blob/master/fastly/fastly_test.go#L19)
3. delete the recording in `fixtures` you want to renew
4. run the test: i.e. `$ go test -timeout 30s -run TestClient_GetDictionaryInfo `
5. replace the IDs of your dummy service with `7i6HN3TK9wS159v2gPAZ8A` in all recordings manually 😭
6. make sure that the API token is not exposed in the recordings otherwise, replace it with `xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx` in all recordings manually 😭
7. put `7i6HN3TK9wS159v2gPAZ8A` back in [here](https://github.com/fastly/go-fastly/blob/master/fastly/fastly_test.go#L19)
8. push your changes to the PR
9. ping me when it's ready to review 😃

Reply:
Green checkmark!  I think it's good to go.
I'd be happy to copy-paste your instructions into a different docs PR if that would help 😂